src/backend/columnar/sql/columnar--11.0-2--11.1-1.sql

```diff
@@ -0,0 +1,28 @@
+-- rename columnar schema to columnar_internal and tighten privileges
+
+REVOKE ALL PRIVILEGES ON ALL TABLES IN SCHEMA columnar FROM PUBLIC;
+ALTER SCHEMA columnar RENAME TO columnar_internal;
+REVOKE ALL PRIVILEGES ON SCHEMA columnar_internal FROM PUBLIC;
+
+-- move columnar internal functions from citus schema to columnar_internal
+
+ALTER FUNCTION citus_internal.columnar_ensure_am_depends_catalog()
+  SET SCHEMA columnar_internal;
+ALTER FUNCTION citus_internal.upgrade_columnar_storage(regclass)
+  SET SCHEMA columnar_internal;
+ALTER FUNCTION citus_internal.downgrade_columnar_storage(regclass)
+  SET SCHEMA columnar_internal;
+
+-- create columnar schema with public usage privileges
+
+CREATE SCHEMA columnar;
+GRANT USAGE ON SCHEMA columnar TO PUBLIC;
+
+-- create view for columnar table options
+
+CREATE VIEW columnar.options AS
+  SELECT regclass, chunk_group_row_limit, stripe_row_limit, compression, compression_level
+    FROM columnar_internal.options
+    WHERE has_table_privilege(regclass, 'SELECT');
```

Comment:
would it be possible to create a similar view for the other tables?

Reply:
I added `security_barrier` views for all internal data. They are actually better than the raw tables, because they also include the relation (before it just included the storage ID, which was awkward to use).